### ISSUE TITLE
space event handler/hook doesnt work (didnt check others)

### ISSUE THREAD

**Issue by codepreneur** [terrawheat/ember-cli-keyboard-actions#1]
For example:

```
export default Ember.Component.extend({

keyUpActions: {
    space: function() {
      console.log('space up clicked!'); // => it never fires
      this.send('toggleCheckbox');
      return false;
    }
  },

  keyDownActions: {
    space: function() {
      console.log('space down clicked!'); // => it never fires
      return false;
    }
  },

  actions: {
    toggleCheckbox() {
      if (!this.disabled) {
        this.toggleProperty('checked');
      }
    }
  }

});
```

**Comment by terrawheat** [comment]
Thanks for raising this, I'll look in to this asap.

**Comment by terrawheat** [comment]
@codepreneur fixed and available on NPM as v0.3.1
